app/src/ui/diff/syntax-highlighting/index.ts

```diff
@@ -118,7 +118,7 @@ export async function getFileContents(
       : Promise.resolve(null)
 
   const newContentsPromise =
-    enableTextDiffExpansion() || lineFilters.newLineFilter.length
+    !repo.codespace && (enableTextDiffExpansion() || lineFilters.newLineFilter.length)
```

Comment:
Pretty sure this is a hack to try and avoid some disk operation. Need to find a real way to handle various file access stuff that happens here if we want to try and mainline this.